src/main/java/io/github/syst3ms/skriptparser/util/StringUtils.java

```diff
@@ -303,4 +303,43 @@ else if (plural)
                 return "a " + noun;
         }
     }
+
+    public static String camelCase(String str, boolean firstNoCase) {
+        String[] parts = str.split(" ");
+        StringBuilder ret = new StringBuilder();
+        for (String part : parts) {
+            if (firstNoCase) {
+                firstNoCase = false;
+                ret.append(part.toLowerCase());
+            } else {
+                ret.append(part.substring(0, 1).toUpperCase()).append(part.substring(1).toLowerCase());
```

Comment:
Couldn't the first part just use charAt and then Character.toUpperCase? That feels a bit more elegant.